ItemPool.py

```diff
@@ -963,6 +963,13 @@ def get_pool_core(world: World) -> tuple[list[str], dict[str, Item]]:
         # Update pattern matcher since all normal junk is removed.
         item_groups['Junk'] = remove_junk_ludicrous_items
         world.distribution.distribution.search_groups['Junk'] = remove_junk_ludicrous_items
+        #Precomplete Dungeons with Ludicrious Item Pool
+        if world.settings.empty_dungeons_mode != 'none':
+            precomplete_locations: list[str] = [location in world.empty_dungeons]
+            for location in precomplete_locations:
+                pool.remove(random.choice(duplicate_items))
+            for location in precomplete_locations:
+                pool.append(random.choice(remove_junk_ludicrous_items))
```

Comment:
Does this need to be 2 separate loops or can it just be 1 loop?

Reply:
The reason I had it in 2 loops was so it didn't try and remove an item added in the previous loop, but given it still not working in your test I think I might need to try something different. I think the issue is that it's filling the extra junk anywhere instead of the precomplete dungeon first.